pkg/kv/kvserver/store_snapshot.go

```diff
@@ -710,6 +710,7 @@ func (s *Store) receiveSnapshot(
 	if err != nil {
 		return err
 	}
+	s.metrics.RangeSnapshotRcvdBytes.Inc(header.RangeSize)
```

Comment:
so I don't really know this area of the codebase well so maybe this is nonsensical, but should we wait to inc this until after we've processed it below (e.g. do this on 724)?

Reply:
The only distinction is that we're currently measuring it once it's received instead of when it's applied (which is generally not an interesting distinction, unless I'm missing something). 

I've now pushed this metrics collection to inside the method that actually reads each chunk from the network. This should make it such that deriving this new metric would act as a rough proxy for the network bandwidth consumed by snapshots.

Reply:
> generally not an interesting distinction

I was wondering if since it was a stream, inc'ing by the whole amount up front could overcount if we hit an error and didn't actually end up pulling it all off the stream? but either way, doing it per-chunk as we consume sounds great.